runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java

```diff
@@ -100,7 +100,7 @@ public WrapperHelper compile(Class<?> wrapperType, Method[] setMethods,
                     try {
                         cls = Thread.currentThread().getContextClassLoader().loadClass(newClassName);
                     } catch (ClassNotFoundException e) {
-                        LOG.warning("Wrapper helper class not found : " + e.toString());
+                        LOG.fine("Wrapper helper class not found : " + e.toString());
```

Comment:
I missed that myself elsewhere (need to send a PR for that some day), but we should use the "formatting" variant wherever possible, meaning `.debugf("Wrapper helper class not found : %s", e)`.

Reply:
Agreed, I think we need a general PR to address much of the logging throughout the code; the intent with this PR was just the minor log level change.